packages/terra-popup/src/PopupContent.scss

```diff
@@ -7,7 +7,7 @@
     }
 
     .rounded-corners {
-      border-radius: 5px;
+      border-radius: var(--terra-popup-content-rounded-corners-border-radius, 3px);
```

Comment:
@msaikaushik1992 lets convert to rems since we have started using them for other border-radiuses in other components: 0.3571rem

Reply:
This should not be themed, and changing the value from 5 to 3 should also not be done. Internal logic is dependent on this value being static.